tests/test_hcp.py

```diff
@@ -57,7 +58,7 @@ def test_test_connection_without_mounting_bucket() -> None:
 
 def test_list_objects() -> None:
     test_mount_bucket()
-    assert type(hcp_h.list_objects()) == list
+    assert type(list(hcp_h.list_objects())) == list
```

Comment:
Wont this just test that the typecasting to list works?

Reply:
Since `list_objects` will return a Python generator instead of a `list`, we need to cast it to `list`. If that casting fails (or any exceptions are raised within `list_objects`) assertion will fail and thus the test also fails